app/Http/Controllers/HomeController.php

```diff
@@ -24,14 +27,16 @@ public function __construct()
     public function index()
     {
         $unreadBook = (session('disable_book_suggestion')) ? null : Book::getRandomUnreadBook();
+        $centre = OfficeLocation::orderBy('centre_name', 'asc')->get();
 
-        return view('home')->with(['book' => $unreadBook]);
+        return view('home')->with(['book' => $unreadBook, 'centre' => $centre]);
```

Comment:
Please update the all places.
```suggestion
        return view('home')->with(['book' => $unreadBook, 'centres' => $centres]);
```